test/e2e/dnslookup_test.go

```diff
@@ -130,20 +133,32 @@ func TestMain(m *testing.M) {
 	err = writeDNSHammerFile(filepath.Join(tmpDir, dnsHammerCNAMERecords), cnameRecords)
 	requireNoErrorf(err, "Failed generating CNAME test data")
 
-	logrus.Infof("Dowloading %v wsl distro tarball", wslTarballName)
-	tarballPath := filepath.Join(tmpDir, wslTarballName)
-
-	err = downloadFile(tarballPath, wslTarballURL)
-	requireNoErrorf(err, "Failed to download wsl distro tarball: %v", err)
+	tarballPath := filepath.Join(os.TempDir(), wslTarballName)
+	s, err := os.Stat(tarballPath)
+	// Only download the tarball if
+	// 1) it doesn't exist
+	// 2) the cache is expired > 60 min
```

Comment:
Why do you care about this? The file should never change, so why would it expire?